pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BacklogQuotaManager.java

```diff
@@ -220,7 +220,7 @@ private void dropBacklogForTimeLimit(PersistentTopic persistentTopic, BacklogQuo
                     }
                     // Timestamp only > 0 if ledger has been closed
                     if (ledgerInfo.getTimestamp() > 0
-                            && currentMillis - ledgerInfo.getTimestamp() > quota.getLimitTime()) {
+                            && currentMillis - ledgerInfo.getTimestamp() > quota.getLimitTime() * 1000) {
```

Comment:
It's a confusing name :)

We should deprecate the old name `limitTime` and introduce a new one `limitTimeInSec`
Not for this PR, we can start a new discussion in the dev mailing list and make it only happen on the next major release